src/main/java/de/zalando/aruha/nakadi/service/EventStreamConfig.java

```diff
@@ -13,7 +13,7 @@
     private static final int BATCH_LIMIT_DEFAULT = 1;
     private static final int STREAM_LIMIT_DEFAULT = 0;
     private static final int BATCH_FLUSH_TIMEOUT_DEFAULT = 30;
-    private static final int STREAM_TIMEOUT_DEFAULT = 60;
+    private static final int STREAM_TIMEOUT_DEFAULT = 0;
```

Comment:
No, I insist that we must remove it. It's in the API and users were notified that it was a temporary solution.